docs/types.md

```diff
@@ -112,10 +112,23 @@ Enums are a closed set of possible string values. Failing to provide a value wit
 
 ## Padding
 
-Paddings are similar to [CCS' padding object](https://developer.mozilla.org/en-US/docs/Web/CSS/padding) where you can specify padding in all directions using a single number, top-bottom left-right using two numbers and top, right, bottom, left using four numbers.
+!!! note
```

Comment:
Does this look better in mkdocs? Currently it doesn't look good when viewed in github.

I suggest to create the regular sdk support table beneath this type and specify the support matrix as usual, like we did in the sprite.md file.

Reply:
Support for asymmetric padding was added in maplibre 2.2, in the following PR:
https://github.com/maplibre/maplibre-gl-js/pull/1289
So a table would be the ideal solution here...

Reply:
OK.

Reply:
Do you want to use a macro for the support table?

Copying and duplicating it all over the place might not be a very sustainable solution.

Reply:
I don't want to complicate things, while I don't like copy pasting stuff, I find it practical for this purpose.